lib/action_subscriber/march_hare/subscriber.rb

```diff
@@ -7,6 +7,12 @@ def cancel_consumers!
         march_hare_consumers.each(&:cancel)
       end
 
+      def create_queue(channel, queue_name, queue_options)
+        queue = ::MarchHare::Queue.new(channel, queue_name, queue_options)
+        queue.declare!
```

Comment:
those integration tests just saved my bacon again! It turns out that building a queue explicitly like this doesn't automatically declare it with the broker the way `channel.queue` does. 🌈